src/component-systems/ClippingSystem.js

```diff
@@ -1,24 +1,29 @@
 import System from '../core/System';
 
 /**
- *
+ * This system supports 3D clipping following threejs's clipping planes setup.
+ * See https://threejs.org/docs/?q=material#api/en/materials/Material.clippingPlanes for more information.
  */
 export class ClippingSystem extends System {
     /**
-     *
+     * ClippingSystem's default constructor
+     * // TODO - add more info
      */
     constructor() {
         super(false);
+        // TODO - how would you describe these vectors in relation to the clipping planes?
```

Comment:
@michaelthatsit - quick question for you

Reply:
these are coplanar points. My understanding is they're needed to calculate the orientation and position if the plane. They're here to be reused for every plane so we're not generating a ton of vectors to be garbage collected.